susemanager-utils/susemanager-sls/salt/packages/init.sls

```diff
@@ -15,7 +15,7 @@ mgr_install_products:
 {%- endif %}
 {%- endif %}
 
-{%- if grains['os_family'] == 'Suse' and grains['instance_id'] is defined and "opensuse" not in grains['oscodename']|lower%}
+{%- if grains['os_family'] == 'Suse' and grains['instance_id'] is defined and "openSUSE" not in grains['oscodename'] %}
```

Comment:
We can give it a try. I hope that the code name will not change in the near future:-)

Alex said, that the "grains['instance_id'] is defined" might not be correct either.
`'instance_id' in grains` maybe together with a "has a value" check would be more correct.
Not sure if you want to include such a change in this as well

Reply:
If you give me it as a suggestion code here, I can apply the suggestion. Otherwise, I will let it out, because I'm not sure how to code it.

Reply:
go with this now.

Reply:
It's okay like that since that code does return an undefined object:
> `foo['bar']` works mostly the same with a small difference in sequence:
>
>    check for an item `'bar'` in _foo_. (`foo.__getitem__('bar')`)
>
>    if there is not, check for an attribute called _bar_ on _foo_. (`getattr(foo, 'bar')`)
>
>    if there is not, return an undefined object.

[Source](https://jinja.palletsprojects.com/en/3.0.x/templates/#variables)